src/Services/RestfulService.php

```diff
@@ -88,13 +111,13 @@ public function patch($model, $data)
      */
     public function persistResource(RestfulModel $resource) {
         try {
-            $resource->save();
+            $resource->update($data);
```

Comment:
I don't think update() is what should be ran on a new resource ? If it doesn't exist, how will eloquent be able to update it?